src/server/pfs/s3/bucket.go

```diff
@@ -17,7 +16,7 @@ import (
 )
 
 func newContents(fileInfo *pfsClient.FileInfo) (s2.Contents, error) {
-	t, err := types.TimestampFromProto(fileInfo.Committed)
+	t, err := permissiveTimestampFromProto(fileInfo)
```

Comment:
I have a slight style preference for just inlining this function. In fact, looking at the code of `TimestampFromProto`, I wonder if we should just be ignoring these errors. It explicitly treats a nil timestamp as the zero time, and the only other errors are from out-of-range dates which shouldn't be an issue since we are the ones generating the timestamps.